DOC-4280: Document the properties of sort in People directory (#88)
exoplatform/exo-documentation: docs/Configuration.rst

    -  :ref:`User inactivity delay configuration <UserInactivityDelay>`
       How to define the user inactivity delay.

    -  :ref:`Sort field configuration <Configuration.sortField>`

>        How to define the on which make the sort in :ref:`People directory <People_Directory>`.

Reviewer: ```suggestion
       How to define sort order in :ref:`People directory <People_Directory>`.
```

     # The delay when we consider a user as offline. Default value is 240000 milliseconds
    exo.user.status.offline.delay=240000
    
.. _Configuration.sortField:    
    
=========================    
Sort field configuration 
=========================

> In :ref:`people application <People_Directory>` users are sorted to facilitate 

Reviewer: ```suggestion
In :ref:`people application <People_Directory>`, users are sorted to facilitate 
```

the search in the directory.

> It is possible to choose the field on which to make the sort by defining this 

Reviewer: ```suggestion
It is possible to choose the field on which to sort by defining this 
```

parameter in :ref:`exo.properties <Configuration.ConfigurationOverview>` file:

::

	exo.social.identity.sort.field=fullname
	

> .. note:: The sort is feasible on these fields: fullname, lastname or firstname.

Reviewer: ```suggestion
.. note:: The following values are accepted: fullname, lastname or firstname.
```

Reviewer: I think we should explain where the full name is picked from. How different is it form what is displayed ?

Author: done

> You can also define the sort direction; ASC or DESC by defining this parameter 

Reviewer: ```suggestion
You can also define the sort direction (ASC for ascending or DESC for descending) with this parameter 
```

in :ref:`exo.properties <Configuration.ConfigurationOverview>` file:

:: 

	exo.social.identity.sort.direction=asc
	
By default, users are sorted in the :ref:`people directory <People_Directory>`

> by their fullnames with ascending order.

Reviewer: ```suggestion
by their full names in ascending order.
```
